source/configuration_manual/shared_mailboxes/cluster_setup.rst

```diff
@@ -35,6 +35,13 @@ to access shared mailboxes of users which live on other backends via the IMAP
 protocol. For this purpose shared mailboxes in a cluster must be accessed via
 another namespace which has ``location`` configured to use ``imapc``.
 
+It is recommended to disable shared namespaces for master users so you should
+add ``userdb_namespace/shared/disabled=yes`` to your master user database as
+:ref:`extra field <authentication-user_database_extra_fields>`. Depending on
```

Comment:
This is a bit confusing. It uses "userdb_" prefix so it indicates that it should be set in passdb, but you don't specifically mention passdb and the link is also pointing to userdb extra fields, not passdb extra fields.